js/oceanex.js

```diff
@@ -0,0 +1,323 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ExchangeError } = require ('./base/errors');
+
+module.exports = class oceanex extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'oceanex',
+            'name': 'oceanex',
+            'countries': [ 'SG' ],
+            'urls': {
+                'api': 'https://api.oceanex.pro/v1/',
+                'www': 'https://www.oceanex.pro.com',
+                'doc': 'https://api.oceanex.pro/doc/v1/',
+            },
+            'api': {
+                'public': {
+                    'get': [
+                        'markets',
+                        'tickers/{pair}/',
+                        'tickers_multi',
+                        'order_book',
+                        'order_book/multi',
+                        'fees/trading',
+                        'trades',
+                        'timestamp',
+                    ],
+                },
+                'private': {
+                    'get': [
+                        'key',
+                        'members/me',
+                        'orders',
+                        'orders_filter',
+                    ],
+                    'post': [
+                        'orders',
+                        'orders/multi',
+                        'order/delete',
+                        'order/delete/multi',
+                        'orders/clear',
+                    ],
+                },
+            },
+            'version': 'v1',
+            'rateLimit': 1000,
+            'has': {
+                'fetchMarkets': true,
+                'fetchCurrencies': false,
+                'fetchTicker': true,
+                'fetchOrderBook': true,
+                'fetchTrades': true,
+                'fetchTradingLimits': false,
+                'fetchTradingFees': true,
+                'fetchFundingFees': false,
+                'fetchTime': true,
+                'fetchOrders': true,
+                'fetchBalance': true,
+                'createOrder': true,
+                'cancelOrder': true,
+                'cancelAllOrders': true,
+            },
+            'timeframes': {
+                '1m': '1m',
+                '5m': '5m',
+                '15m': '15m',
+                '30m': '30m',
+                '1h': '1h',
+                '4h': '4h',
+                '12h': '12h',
+                '1d': '1d',
+                '1w': '1w',
+            },
+        });
+    }
+
+    async fetchMarkets (params = {}) {
+        let response = await this.publicGetMarkets ();
```

Comment:
We suggest that to use `const` for immutables (if the value doesn't change) everywhere throughout the file.